grafana-dashboard.tf

```diff
@@ -1,36 +1,37 @@
 locals {
   graph_id_base = 100
-  apiary_data_buckets = 2
+  number_of_buckets = 2
```

Comment:
Isn't this actually `number_of_graphs_per_bucket`?  I thought that is what this is being used for.

Reply:
Yes it is!